src/configMeta.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import type { ConfigValue, ConfigPropertyMeta } from '@salesforce/core';
+
+export enum ConfigVars {
```

Comment:
```suggestion
export enum const ConfigVars {
```
https://www.typescriptlang.org/docs/handbook/enums.html#const-enums are cleaner almost always.